.github/workflows/atd_moped_api.yml

```diff
@@ -22,13 +22,13 @@ jobs:
     name: Build
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
           python-version: "3.8"
```

Comment:
this property is [now required](https://github.com/actions/setup-python/releases/tag/v4.0.0) with `v4` - fortunately we were already using it :)